Update ome-advanced-message-encryption.md (#7949)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/ome-advanced-message-encryption.md

You can only revoke and set an expiration date for emails sent to external recipients.

> Note:"Advanced Message Encryption revocation and expiration functions will not function unless the "Premium Encryption in Office 365" option is enabled in their E5 license."

Reviewer: ```suggestion
> [!NOTE]
> The Advanced Message Encryption revocation and expiration functions will not work unless you enable the **Premium Encryption in Office 365** option in your E5 license.
```
